test/BiochemistryAPI_server_test.py

```diff
@@ -151,7 +139,7 @@ def test_search_compounds(self):
         self.assertEqual(len(res), 1)
         self.assertEqual(res[0]["id"], "cpd00007")
         res = self.getImpl().search_compounds(self.ctx, {"query": "O2"})[0]
-        self.assertEqual(len(res), 3)
```

Comment:
Seems like these might change according to the database too?

Reply:
exactly